src/components/pages/ZaboPage/ZaboPage.js

```diff
@@ -8,6 +8,10 @@ import ZaboList from 'templates/ZaboList';
 import Header from 'templates/Header';
 import Carousel from 'react-airbnb-carousel';
 
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.bubble.css';
+import './quill.viewer.scss';
```

Comment:
Use styled-component instead.
You can add required styles in ZaboPageWrapper.Info.Body
quill.bubble.css also

Reply:
Moved quill.viewer.scss only